tasks/99_fat-fingers/fat-fingers.js

```diff
@@ -0,0 +1,12 @@
+export const fatFingers = (str) => {
+  const arrayOfString = str.split(/[Aa]/);
+  return arrayOfString.map((cur, ind) => {
```

Comment:
you can use reduce here and it will allow for you omit first check and join function in the end